src/components/Chat/InputMessageV2.tsx

```diff
@@ -142,11 +329,11 @@ const InputMessageV2 = ({
   const sendButtonStyle = React.useCallback(() => {
     const isDisabled = isDisableButton();
     if (isDisabled) return colors.gray1;
-    if (type === 'SIGNED') return colors.darkBlue;
+    if (type === SIGNED) return colors.darkBlue;
     return colors.bondi_blue;
   }, [isDisableButton()]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendButtonStyle` function has been updated to use constants for comparison instead of string literals. This is a good practice as it reduces the chance of typos causing bugs.



<!-- This is an auto-generated comment by OSS CodeRabbit -->